pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -251,6 +279,21 @@ func (csh *cisScanHandler) Updated(cs *v3.ClusterScan) (runtime.Object, error) {
 			}
 		}
 
+		// Wait for complete pod removal
+		removed := false
+		for retry := NumberOfRetriesForPodRemoval; retry > 0; retry-- {
+			time.Sleep(RetryIntervalInMilliseconds * time.Millisecond)
+			podName := fmt.Sprintf("security-scan-runner-%v", cs.Name)
+			_, err := csh.podLister.Get(v3.DefaultNamespaceForCis, podName)
+			if err != nil && kerrors.IsNotFound(err) {
```

Comment:
return if err is !notFound

Reply:
will refactor this duplicated logic into a diff function